auth/pom.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.sixsq.slipstream</groupId>
+  <artifactId>auth</artifactId>
+  <packaging>jar</packaging>
+  <version>2.19-SNAPSHOT</version>
+  <name>auth</name>
+  <description>Authentication Service</description>
+  <url>http://sixsq.com</url>
+
+  <parent>
```

Comment:
This should inherit from its immediate parent, not directly from the main `pom.xml` file.